src/fizzBuzz.js

```diff
@@ -0,0 +1,23 @@
+'use strict'
+
+export default function fizzBuzz() {
+  var arrayNums = [];
+  for (var i = 1; i < 101; i++){
+    arrayNums.push(i)
+  }
+
+  for(var i = 1; i < arrayNums.length; i++) {
+    if (arrayNums[i] % 3 == 0 ){
```

Comment:
it would be better to use the ```===``` comparison operator 
check out this reference if you would like
http://stackoverflow.com/questions/359494/which-equals-operator-vs-should-be-used-in-javascript-comparisons